src/main/java/uk/gov/hmcts/reform/sscs/service/SendNotificationService.java

```diff
@@ -80,22 +80,31 @@ public SendNotificationService(
         this.pdfLetterService = pdfLetterService;
     }
 
-    void sendEmailSmsLetterNotification(
+    boolean sendEmailSmsLetterNotification(
             NotificationWrapper wrapper,
             Notification notification,
             SubscriptionWithType subscriptionWithType,
             NotificationEventType eventType) {
-        sendEmailNotification(wrapper, subscriptionWithType.getSubscription(), notification);
-        sendSmsNotification(wrapper, subscriptionWithType.getSubscription(), notification, eventType);
+        boolean emailSent = sendEmailNotification(wrapper, subscriptionWithType.getSubscription(), notification);
+        boolean smsSent = sendSmsNotification(wrapper, subscriptionWithType.getSubscription(), notification, eventType);
 
         boolean isInterlocLetter = INTERLOC_LETTERS.contains(eventType);
         boolean isDocmosisLetter = DOCMOSIS_LETTERS.contains(eventType);
+        boolean letterSent = false;
         if ((lettersOn && !isInterlocLetter) || (interlocLettersOn && isInterlocLetter) || (docmosisLettersOn && isDocmosisLetter)) {
-            sendLetterNotification(wrapper, subscriptionWithType.getSubscription(), notification, subscriptionWithType, eventType);
+            letterSent = sendLetterNotification(wrapper, subscriptionWithType.getSubscription(), notification, subscriptionWithType, eventType);
         }
+
+        boolean notificationSent = emailSent | smsSent | letterSent;
+
+        if (!notificationSent) {
+            log.error("Did not send a notification for event {} for case id {}.", eventType.getId(), wrapper.getCaseId());
```

Comment:
are you sure you need another error log here?

Reply:
The purpose of this ticket is to increase readability when tracing issues through the logs, therefore this is required.

Reply:
Chatted to Thor. It will increment the error count by one. He says it happens all the time amongst different apps logging the same error, and doesn't mind this.